src/plugins/run/blockquote.ts

```diff
@@ -0,0 +1,14 @@
+import * as unified from 'unified';
+import { visit } from 'unist-util-visit';
+
+import { MDAST } from '../../ast-types';
+
+export const blockquoteMdast2Hast: unified.Attacher = function () {
+  return function transform(root) {
+    visit<MDAST.Blockquote>(root, 'blockquote', node => {
+      const data = node.data || (node.data = {});
```

Comment:
```suggestion
      const data = node.data ||= {};
```
if you feel fancy :see_no_evil:

Reply:
I always feel fancy